src/containers/Login/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button, Form, Input } from 'antd';
+import { LockOutlined, UserOutlined } from '@ant-design/icons';
+
+import styles from './style.module.css';
+
+const { Password } = Input;
+const { Item: ItemForm } = Form;
+
+const roleRequired = (message) => {
+  return{
+    required: true,
+    message,
+  }
+}
+
+const LoginContainer = ({ onSubmit }) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.wrapper}>
+        <h1>Estoque - RP</h1>
+        <img src="../../retina.png" />
```

Comment:
Faltou adicionar o alt da image sem ele o console fica imprimindo um alerta 
![image](https://user-images.githubusercontent.com/20932604/101106675-5a920500-35af-11eb-9030-dc3eb33b4721.png)

```suggestion
        <img src="../../retina.png"  alt="company logo"/>
```